Packs/GoogleChronicleBackstory/ReleaseNotes/3_0_0.md

```diff
@@ -0,0 +1,50 @@
+
+#### Classifiers
+
+##### Chronicle - Classifier
+
+- Updated the classifier for newly added **Chronicle Curated Rule Detection** incident type.
+
+#### Incident Fields
+
+- **Chronicle Detection Window End Time**
+- **Chronicle Rule ID**
+- **Chronicle Detection ID**
+- **Chronicle Detection State**
+- New: **Chronicle Curated Events**
```

Comment:
Is this the only one that is new?  Perhaps move it to the end of the list of Incident Fields.